spec/support/protos/google_unittest.proto

```diff
@@ -663,26 +829,24 @@ message TestParsingMerge {
     repeated TestAllTypes field1 = 1;
     repeated TestAllTypes field2 = 2;
     repeated TestAllTypes field3 = 3;
-    // TODO: Support groups
-    //repeated group Group1 = 10 {
-    //  optional TestAllTypes field1 = 11;
-    //}
-    //repeated group Group2 = 20 {
-    //  optional TestAllTypes field1 = 21;
-    //}
+    repeated group Group1 = 10 {
+      optional TestAllTypes field1 = 11;
+    }
+    repeated group Group2 = 20 {
+      optional TestAllTypes field1 = 21;
+    }
     repeated TestAllTypes ext1 = 1000;
     repeated TestAllTypes ext2 = 1001;
   }
   required TestAllTypes required_all_types = 1;
   optional TestAllTypes optional_all_types = 2;
   repeated TestAllTypes repeated_all_types = 3;
-  // TODO: Support groups
```

Comment:
Hmmm. Strange. It is able to read the proto and the generated code looks right. Lemme dig up the protobuf docs on group.

Leaving it as is. It seems to work fine.